ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -124,6 +130,9 @@ class PhaserUnit extends PhaserAnimatedEntity {
 		attrs: AttributeData[]
 	}): void {
 		console.log('PhaserUnit render-attributes', data); // TODO remove
+		const attributesContainer = this.attributesContainer = this.scene.add.container(0,
+			25 + Math.max(this.sprite.displayHeight, this.sprite.displayWidth) / 2);
```

Comment:
as we discussed before, this value is better to be calculated as average of displayHeight/2 and displayWidth/2 so that in case unit sprite is five times wider than long, attributes will not be offset with a huge gap